README.md

```diff
@@ -3,12 +3,7 @@
 # OpenHTF
 The open-source hardware testing framework.
 
-[![Build Status](https://travis-ci.org/google/openhtf.svg?branch=master)](https://travis-ci.org/google/openhtf)
```

Comment:
This works both in my browser (redirects to https://api.travis-ci.org/google/openhtf.svg?branch=master) and in GitHub. Same with coverage. PR stats only works in my browser, while issue stats works but says Unknown. Can we at least keep the first two if there's no obvious reason for the badges to not work in GitHub or when viewed by others?